app/javascript/hyacinth_ui_v1/components/projects/permissions/PermissionsEdit.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { useParams } from 'react-router-dom';
+
+import TabHeading from '../../ui/tabs/TabHeading';
+import ProjectInterface from '../ProjectInterface';
+import { getProjectQuery } from '../../../graphql/projects';
+import GraphQLErrors from '../../ui/GraphQLErrors';
+import PermissionsEditor from './PermissionsEditor';
+
+function PermissionsEdit() {
+  const { stringKey } = useParams();
+  const { loading, error, data } = useQuery(getProjectQuery, { variables: { stringKey } });
```

Comment:
This project query should load the base project data along with the project permissions. This will reduce the number of queries the PermissionsEditor component has to make. The data can then be passed on to the PermissionEditor. I use this pattern for FieldSets and Publish Targets.

Reply:
Whoops.  Yeah, I definitely should have done that.  Will fix here and in PermissionsShow.js, thanks!